### ISSUE TITLE
ion-toolbar is off/ion-content is hidden behind image

### ISSUE THREAD

**Issue by narhc8** [RaschidJFR/ionic-header-parallax#10]
As you can see from the images, this is what I see after implementing:

![IMG_5637](https://user-images.githubusercontent.com/29511609/61604432-7ccff080-ac0f-11e9-978e-d54fe21724ea.PNG)
![IMG_5636](https://user-images.githubusercontent.com/29511609/61604436-80fc0e00-ac0f-11e9-88f4-2ba7bf135a62.PNG)
![IMG_5635](https://user-images.githubusercontent.com/29511609/61604438-835e6800-ac0f-11e9-9e01-dd6040ed76ab.PNG)

Basically, the ion-toolbar is off so my back button is way up, the contents of ion-contents are hidden behind the image. This might be due to my implementation of setting ion-content to `fullscreen` but I am not sure.

**Comment by narhc8** [comment]
My implementation:

event-details.html:
<img width="615" alt="Screen Shot 2019-07-21 at 11 35 46 PM" src="https://user-images.githubusercontent.com/29511609/61604643-5d859300-ac10-11e9-9f42-2d174e5f674d.png">

event-detail.scss:
<img width="317" alt="Screen Shot 2019-07-21 at 11 35 54 PM" src="https://user-images.githubusercontent.com/29511609/61604650-65ddce00-ac10-11e9-8094-12b714c2b2b2.png">

**Comment by RaschidJFR** [comment]
I’m not sure I’m following you... what’s the expected behavior vs the actual behavior?

**Comment by narhc8** [comment]
The expected behavior is that the content is below the image, but as you can see in the image above the `ion-content` stuff is found to be below the image which is the actual behavior

**Comment by RaschidJFR** [comment]
Mmm... I don't see anything unusual. It looks to me like the parallax color is somehow set to transparent. It's hard to say from just the photos. Can you share it on [stackblitz](https://stackblitz.com/edit/ionic) or similar?

**Comment by RaschidJFR** [comment]
Closing due to inactivity
